cosmotech/data_update_quest_cli/database_dump/redis_dump.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (C) - 2025 - Cosmo Tech
```

Comment:
Why name the folder "database_dump" and the file "redis_dump" ?
You could have made a pure "database" folder, or made a "redis" folder with a "dump" command